woocommerce/woocommerce-gutenberg-products-block
# Cart i2: Render filled and empty Carts on frontend

This PR renders both the filled and empty carts on frontend.

Closes #4792
Closes #4795
## Testing
- Insert Cart i2 into a page.
- Save.
- Visit frontend.
- It should render, just like Cart i1.
- Empty your cart, you should see your empty cart content.
